From 50d50624805b8ca52072fa9f32fb073a2dbc1d9f Mon Sep 17 00:00:00 2001
From: Simeon Marijon <simeon.marijon@foss.st.com>
Date: Tue, 20 Jun 2023 14:36:56 +0200
Subject: [PATCH] arm64: defconfig: add stm32 backup registers support in
 addons fragment

Enable STM32 TAMP backup registers with fragment-02-defconfig-addons.config

Change-Id: Ied55fbb49ae40c176c1149abfa5622596daffdd4
Signed-off-by: Simeon Marijon <simeon.marijon@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -15,6 +15,7 @@ CONFIG_MTD_HYPERBUS=y
 CONFIG_MTD_SPI_NAND=y
 CONFIG_MTD_UBI=y
 CONFIG_NVMEM_STM32_ROMEM=y
+CONFIG_NVMEM_STM32_TAMP=y
 CONFIG_OF_OVERLAY=y
 CONFIG_PHY_STM32_USB2PHY=y
 CONFIG_PWM_STM32=m
